### PR TITLE
Remove Inspection Target

### PR DESCRIPTION
**Pull request by UsamaEquinorAFK**

### CODE REVIEW

**Review by prasm313 — COMMENTED** [review]
LGTM, tested without any bugs. Suggest that @Afonso-2403 takes a look before it gets approved.

**Review by Afonso-2403 — APPROVED** [review]
LGTM, let's not merge yet until we have the workflow in place to update the existing missions

**Pull request merged**
